stack.yaml

```diff
@@ -41,6 +41,8 @@ packages:
 extra-lib-dirs:
     - ./lib/
 
+system-ghc: true
```

Comment:
From my understanding, whether this is needed depends on your setup, so I don't think we should have this as default behavior for everyone.
It seems to work fine for me, so maybe we should figure out the difference of our setup instead

Reply:
The language server crashes with the error `GHC ABIs don't match!`. Adding the above line has fixed it:

https://www.reddit.com/r/haskell/comments/uk5wg7/comment/i7n990t/?utm_source=reddit&utm_medium=web2x&context=3

Reply:
I will help you fix this, but this workaround could cause others setup to break, so please remove this again.

Reply:
Some further digging got me to these workarounds:

https://github.com/haskell/vscode-haskell?tab=readme-ov-file#ghc-abis-dont-match

One workaround is to fix it in your local setup to force stack to always use the global ghc version.
https://www.haskell.org/ghcup/guide/#strategy-2-system-ghc-works-on-all-stack-versions